ui/selectors/selectors.js

```diff
@@ -1884,6 +1884,8 @@ export function getShowPrivacyPolicyToast(state) {
   const currentDate = new Date(Date.now());
   return (
     !newPrivacyPolicyToastClickedOrClosed &&
+    // force e2e tests to use old privacy policy
```

Comment:
NOTE - we probably want to update the tests in the near future to use the new Onboarding UI and Privacy Banner.

Reply:
This does decrease code coverage (new code branch). But I'm guessing we'll remove these env-vars in a future PR once the privacy policy is fully in.

Happy to add some TODO comments, or other approaches if we will clean this up.